.github/dependabot.yaml

```diff
@@ -0,0 +1,16 @@
+version: 2
+updates:
+
+  # GitHub actions
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    reviewers:
+      - "elastic/observablt-ci"
```

Comment:
FWIW, I'd be happy for this to be or to include "elastic/apm-agent-node-js" for reviewing these updates.